packages/extension-browser/src/devtools/panel/utils/inspire.ts

```diff
@@ -22,4 +25,19 @@ const inspire = (): string => {
     return remaining.splice(index, 1)[0];
 };
 
-export default inspire;
+/** Generate a status message that periodically changes. */
+export const useRotatingInspiration = (initialMessage: string) => {
+    const [message, setMessage] = useState(initialMessage);
+
+    useEffect(() => {
+        const interval = setInterval(() => {
+            setMessage(inspire());
+        }, 7500);
+
+        return () => {
+            clearInterval(interval);
+        };
+    });
+
+    return message;
+};
```

Comment:
Doesn't eslint complain about not ending empty line? I thought we had a rule for that 🤔

Reply:
Looks like there already is an empty line at the end of this file...